ckanext/switzerland/helpers/validators.py

```diff
@@ -212,10 +212,20 @@ def validator(key, data, errors, context):
 @scheming_validator
 def ogdch_license_required(field, schema):
     def validator(key, data, errors, context):
-        value = data[key]
-        if value is missing or value is None:
-            rights = data.get(key[:-1] + ('rights',))
+        license = data[key]
+        if license not in (missing, None):
+            data[key] = license
+            return
```

Comment:
would be nice to have some some logs for the inner function.

Do we want to describe also a scenario that both  license and  rights are present?
Or we just go from top and this is enough?

Reply:
This is enough. If there is a license, we use the license. It's only if there is not a license that we try to use the rights.

Reply:
ok, thanks, fine with me